frontend/src/components/pages/profile/ProfileCard/variants/OrgCard/OrgAction.tsx

```diff
@@ -0,0 +1,87 @@
+import OrgListItem from "./OrgListItem";
+import { useState } from "react";
+import Link from "next/link";
+import { ProfileActionProps } from "@components/pages/profile/ProfileCard/variants/ProfileCardBase";
+import { useRouter } from "next/router";
+
+const orgLink = (orgId: string) => `/orgs/${orgId}`;
+
+export type Props = {
+  orgs: Pick<Organization, "id" | "name" | "logoUrl">[];
+};
+
+/**
+ * Function that returns a functional component for use as a profile card action.
+ * The outer function takes the props not required by the card action.
+ * The inner function takes the remaining props as required for the card.
+ *
+ * This function takes a list of orgs, and returns different components depending on its length:
+ * - No orgs: Text
+ * - 1 org: List to its org admin page
+ * - Multiple orgs: Menu with links to their admin pages
+ */
+export const createOrgProfileAction = ({ orgs }: Props): React.VFC<ProfileActionProps> => {
+  const OrgProfileAction: React.VFC<ProfileActionProps> = (actionProps) => {
+    switch (orgs.length) {
+      case 0:
+        return (
+          <Box fontStyle="italic">
+            <CardActions>
+              <Typography variant="overline" color="textSecondary">
+                Ingen organisasjoner
+              </Typography>
+            </CardActions>
+          </Box>
+        );
+      case 1:
+        return (
+          <CardActionArea>
+            <Link href={orgLink(orgs[0].id)} passHref>
+              <CardActions data-test-id={actionProps["data-test-id"]}>
+                <OrgListItem org={orgs[0]} />
+              </CardActions>
+            </Link>
+          </CardActionArea>
+        );
+      default:
+        return <OrgList orgs={orgs} {...actionProps} />;
+    }
+  };
+
+  return OrgProfileAction;
+};
+
+const OrgList: React.VFC<Props & ProfileActionProps> = ({ orgs, "data-test-id": dataTestId }) => {
+  const [menu, setMenu] = useState<{ open: boolean; anchor: Element | undefined }>({ open: false, anchor: undefined });
+  const router = useRouter();
+
+  return (
+    <>
+      <CardActionArea>
+        <CardActions
+          onClick={(event) => setMenu({ open: true, anchor: event.currentTarget })}
+          data-test-id={`${dataTestId}link`}
+        >
+          <Typography variant="overline" color="textPrimary">
+            Se organisasjoner
+          </Typography>
+        </CardActions>
+      </CardActionArea>
+      <Menu
+        open={menu.open}
+        onClose={() => setMenu({ open: false, anchor: undefined })}
+        anchorEl={menu.anchor}
+        anchorOrigin={{ vertical: "bottom", horizontal: "left" }}
+        transformOrigin={{ vertical: "bottom", horizontal: "left" }}
+      >
+        {orgs.map((org) => (
+          <MenuItem key={org.id} onClick={() => router.push(orgLink(org.id))}>
+            <OrgListItem org={org} />
+          </MenuItem>
+        ))}
+      </Menu>
```

Comment:
Maybe this could be a more aesthetically pleasing component? I envision an accordion-button that reveals a list of organizations making the card taller. This might be too much additional work for this PR.

Reply:
Agreed. I struggled to find a way for the menu to be separated enough from the background, as our range of background colors seem limited, so whatever color I picked it merged with some background.

Not sure if accordion is the way to go though, since moving things around on the page might negatively affect user experience?

Another solution may be to keep the `/orgs` page for when you have more than 1 org.

Reply:
Piggybacking on this: Would it be possible to replace the undraw-illustration with the organization logo instead having an additional image on the card? As for keeping `/orgs`: it also opens up the possibility for us to do more with that page, e.g. make it the go-to place for creating new organizations. I agree with @hermannm's concerns regarding an accordion, and also with @jorul's concerns regarding the dropdown. However, my concern could likely be attributed to a general concern regarding the design of dropdowns.